### PR TITLE
feature(core) Default messenger configuration per module

### PR DESCRIPTION
**Pull request by piotrkreft**
# Description

Default messenger configuration per module

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Unit Tests
- [ ] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: module/category/src/Resources/config/messenger.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+framework:
+    messenger:
+        routing:
+            Ergonode\Category\Domain\Command\CreateCategoryCommand: '%ergonode.designer.messenger_transport_name%'
```
Comment (wfajczyk) [inline module/category/src/Resources/config/messenger.yaml:4]:
```suggestion
            Ergonode\Category\Domain\Command\CreateCategoryCommand: '%ergonode.category.messenger_transport_name%'
```

File: module/category/src/Resources/config/messenger.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+            Ergonode\Category\Domain\Command\UpdateCategoryCommand: '%ergonode.designer.messenger_transport_name%'
```
Comment (wfajczyk) [inline module/category/src/Resources/config/messenger.yaml:5]:
```suggestion
            Ergonode\Category\Domain\Command\UpdateCategoryCommand: '%ergonode.category.messenger_transport_name%'
```

File: module/mailer/src/Resources/config/messenger.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+framework:
+    messenger:
+        routing:
+            Ergonode\Mailer\Domain\Command\SendMailCommand: '%ergonode.mailer.messenger_transport_name%'
```
Comment (wfajczyk) [inline module/mailer/src/Resources/config/messenger.yaml:4]: There are no `transports` to mail.

File: module/notification/src/Resources/config/messenger.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+framework:
+    messenger:
+        transports:
+            notification: '%ergonode.notification.messenger_transport%'
```
Comment (wfajczyk) [inline module/notification/src/Resources/config/messenger.yaml:4]: In this time transport use only to send mail.

**Review by wfajczyk — CHANGES_REQUESTED** [review]

File: module/category/src/Resources/config/messenger.yaml (hunk 1 above)
Comment (piotrkreft) [reply module/category/src/Resources/config/messenger.yaml]: 👍🏻

File: module/category/src/Resources/config/messenger.yaml (hunk 2 above)
Comment (piotrkreft) [reply module/category/src/Resources/config/messenger.yaml]: 👍🏻

File: module/mailer/src/Resources/config/messenger.yaml (hunk 3 above)
Comment (piotrkreft) [reply module/mailer/src/Resources/config/messenger.yaml:4]: there's `notification` and that's the one under parameter

File: module/notification/src/Resources/config/messenger.yaml (hunk 4 above)
Comment (piotrkreft) [reply module/notification/src/Resources/config/messenger.yaml:4]: yup. I'm not altering the behavior, just making it easier and more robust to use

Comment (wfajczyk) [reply module/notification/src/Resources/config/messenger.yaml:4]: Ok. For these moments mailer uses this configuration but does not require this module.

**Review by wfajczyk — APPROVED** [review]

**Review by wiewiurdp — APPROVED** [review]

**Pull request merged**